tests/dao_tests/test_study_nph_dao.py

```diff
@@ -741,6 +744,7 @@ def setUp(self):
         self.nph_site_dao = NphSiteDao()
         self.nph_order_dao = NphOrderDao()
         self.nph_ordered_sample_dao = NphOrderedSampleDao()
+        self.nph_sample_update_dao = NphSampleUpdateDao()
```

Comment:
You could use this DAO on Line 916. Instead of using `database_factory` as shown below. You can replace that with the below code snippet.

```python
with self.nph_sample_update_dao.session() as session:
    result = session.query(SampleUpdate).first()
```

Reply:
Or you can remove the initialization on `Line 747` if you do not want to use it.